esp8266/hspi.c

```diff
@@ -242,34 +246,34 @@ uint32_t spi_transaction(uint8_t spi_no, uint8_t cmd_bits, uint16_t cmd_data,
     if (dout_bits) {
         // Enable MOSI function in SPI module
         SET_PERI_REG_MASK(SPI_USER(spi_no), SPI_USR_MOSI);
-    // Copy data to W0
-    if (READ_PERI_REG(SPI_USER(spi_no))&SPI_WR_BYTE_ORDER) {
-        WRITE_PERI_REG(SPI_W0(spi_no), dout_data << (32 - dout_bits));
-    } else {
-        uint8_t dout_extra_bits = dout_bits%8;
-
-        if (dout_extra_bits) {
-            // If your data isn't a byte multiple (8/16/24/32 bits) and you
-            // don't have SPI_WR_BYTE_ORDER set, you need this to move the
-            // non-8bit remainder to the MSBs. Not sure if there's even a use
-            // case for this, but it's here if you need it... For example,
-            // 0xDA4 12 bits without SPI_WR_BYTE_ORDER would usually be output
-            // as if it were 0x0DA4, of which 0xA4, and then 0x0 would be
-            // shifted out (first 8 bits of low byte, then 4 MSB bits of high
-            // byte - ie reverse byte order).
-            // The code below shifts it out as 0xA4 followed by 0xD as you
-            // might require.
-            WRITE_PERI_REG(SPI_W0(spi_no), (
-                (0xFFFFFFFF << (dout_bits - dout_extra_bits) & dout_data)
-                    << (8-dout_extra_bits) |
-                ((0xFFFFFFFF >> (32 - (dout_bits - dout_extra_bits)))
-                    & dout_data)
-            ));
+        // Copy data to W0
+        if (READ_PERI_REG(SPI_USER(spi_no))&SPI_WR_BYTE_ORDER) {
+            WRITE_PERI_REG(SPI_W0(spi_no), dout_data << (32 - dout_bits));
         } else {
-            WRITE_PERI_REG(SPI_W0(spi_no), dout_data);
+            uint8_t dout_extra_bits = dout_bits%8;
+
+            if (dout_extra_bits) {
+                // If your data isn't a byte multiple (8/16/24/32 bits) and you
+                // don't have SPI_WR_BYTE_ORDER set, you need this to move the
+                // non-8bit remainder to the MSBs. Not sure if there's even a
+                // use case for this, but it's here if you need it... For
+                // example, 0xDA4 12 bits without SPI_WR_BYTE_ORDER would
+                // usually be output as if it were 0x0DA4, of which 0xA4, and
+                // then 0x0 would be shifted out (first 8 bits of low byte,
+                // then 4 MSB bits of high byte - ie reverse byte order).
+                // The code below shifts it out as 0xA4 followed by 0xD as you
+                // might require.
+                WRITE_PERI_REG(SPI_W0(spi_no), (
+                    (0xFFFFFFFF << (dout_bits - dout_extra_bits) & dout_data)
+                        << (8-dout_extra_bits) |
+                    ((0xFFFFFFFF >> (32 - (dout_bits - dout_extra_bits)))
+                        & dout_data)
+                ));
+            } else {
+                WRITE_PERI_REG(SPI_W0(spi_no), dout_data);
+            }
```

Comment:
Please avoid mixing style and functionality changes in one commit. Actually, I'd ask to undo this hunk.